modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -14,6 +14,327 @@
 namespace opencv_test
 {
 
+// FIXME avoid this code duplicate in perf tests
```

Comment:
If you plan to have perf_tests, I suggest to create `gapi_core_tests_common.hpp` file (on the analogy of `gapi_video_tests_common`), put all the stuff there and share it between different tests

Reply:
Fixed. In gapi_core_tests_common.hpp.

Reply:
You didn't actually put it in a new file, you put it in `gapi_tests_common.hpp` which is shared between a-a-all the G-API tests, and I don't really sure if this code can be useful for not-`core`-tests.. But I assume you had offline conversation about that, right?

Reply:
Moved to `gapi_parsers_tests_common.hpp`